invenio_app_rdm/config.py

```diff
@@ -1177,3 +1178,7 @@ class NotificationsUserSchema(UserSchema):
 
 NOTIFICATIONS_SETTINGS_VIEW_FUNCTION = notification_settings
 """View function for notification settings."""
+
+from invenio_rdm_records.services.permissions import RDMRequestsPermissionPolicy
```

Comment:
this is required because we need to tweak the requests permission policy slightly for guest-based access requests